### ISSUE TITLE
Axe DevTools error for Privacy Policy Checkbox

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95538]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

Axe DevTools scan is complaining that the role "checkbox" isn't correct for the label element.

![Image](https://github.com/user-attachments/assets/e85107fb-95cb-4b14-a0a0-b489e49ed052)

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-3
**Page**: review-and-submit
**Environment**: Axe DevTools scan
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [4.1.3 Status Messages Level AA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#status-messages)

### To Reproduce:
- Run Axe DevTools on the Review and Submit page

#### Expected Behavior
-  No aria role error.

#### Actual Behavior
- DevTools scan on the full page shows the error: "ARIA role should be appropriate for the element"

## Tasks
- [ ] Fix error by changing the role or updating the element.
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] No aria role error. Run the scan when checkbox is in checked & unchecked states.
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
